dashboard/next.config.js

```diff
@@ -15,17 +15,20 @@
  *
  */
 
-module.exports = () => {
-  const rewrites = () => {
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  trailingSlash: true,
+
+  rewrites: () => {
```

Comment:
We may use `phase` to enable rewrites conditionally on development builds.

https://nextjs.org/docs/pages/api-reference/next-config-js